src/web/nextui/src/app/api/eval/[id]/route.ts

```diff
@@ -1,107 +0,0 @@
-import { NextRequest, NextResponse } from 'next/server';
-import { validate as uuidValidate } from 'uuid';
-
-import store from '@/app/api/eval/shareStore';
-import { IS_RUNNING_LOCALLY } from '@/constants';
-
-import type { EvaluateTable, FilePath, ResultsFile, UnifiedConfig } from '@/../../../types';
-import { readResult, updateResult, deleteEval } from '@/../../../util';
-
-export const dynamic = IS_RUNNING_LOCALLY ? 'auto' : 'force-dynamic';
-
-async function getDataForId(
```

Comment:
I love the idea behind this PR and it'd be great to delete all this duplicate code (and much healthier for the codebase).  

I believe this route and some of the others are used at app.promptfoo.dev, i.e. the app that serves `promptfoo share` results.